storage/client/Dockerfile

```diff
@@ -2,7 +2,7 @@
 
 # Alpine is chosen for its small footprint
 # compared to Ubuntu
-FROM golang:1.18-alpine
```

Comment:
personally I don't like explicitly defining registry in the dockerfile, since you can do it in your environment and replace with local registry. But it's ok for now and we can address it separately

Reply:
This is due to relatively new behavior from podman.  If you do not specify the cr, for certain image names it will interactively prompt with no timeout.  So automation will never finish.  See https://www.redhat.com/sysadmin/container-image-short-names

Reply:
but you can change `registries.conf` to `permissive` or `disabled` ...